test/libyul/evmCodeTransform/stackReuse/reuse_too_deep_slot.yul

```diff
@@ -33,78 +33,108 @@
 // ====
 // stackOptimization: true
 // ----
-// PUSH1 0x7
-// PUSH1 0x42
-// DUP16
-// PUSH1 0x0
-// SSTORE
-// PUSH1 0x43
-// DUP2
-// PUSH1 0x1
-// MSTORE
-// DUP1
-// PUSH1 0x1
-// MSTORE
-// POP
-// POP
-// DUP15
-// PUSH1 0x1
-// SSTORE
-// DUP14
-// PUSH1 0x1
-// SSTORE
-// DUP13
-// PUSH1 0x1
-// SSTORE
-// DUP12
-// PUSH1 0x1
-// SSTORE
-// DUP11
-// PUSH1 0x1
-// SSTORE
-// DUP10
-// PUSH1 0x1
-// SSTORE
-// DUP9
-// PUSH1 0x1
-// SSTORE
-// DUP8
-// PUSH1 0x1
-// SSTORE
-// DUP7
-// PUSH1 0x1
-// SSTORE
-// DUP6
-// PUSH1 0x1
-// SSTORE
-// DUP5
-// PUSH1 0x1
-// SSTORE
-// DUP4
-// PUSH1 0x1
-// SSTORE
-// DUP3
-// PUSH1 0x1
-// SSTORE
-// DUP2
-// PUSH1 0x1
-// SSTORE
-// DUP1
-// PUSH1 0x1
-// SSTORE
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
-// POP
+//     /* "":15:16   */
+//   0x07
+//     /* "":94:121   */
+//   verbatimbytecode_6042
```

Comment:
Why is this using verbatim and not `push1 0x42`?

Reply:
Because that's what's in the code?

Reply:
But why is it `verbatimbytecode` and not proper push instruction?

Reply:
Why should it be a push instruction - the Yul code has a verbatim instruction :-)! Are you wondering since the old code test setup actually assembled and disassembled instead? That has nothing to do with the code transform - in evmasm it's still verbatim.